### ISSUE TITLE
Event dragging misaligned in vertical mode when Scheduler is scrolled up in absolutely positioned container 

### ISSUE THREAD

**Issue by jsakalos** [bryntum/support#3741]
The showcase is in Angular but the framework has (probably) nothing to do with it.

![dragging-misaligned](https://user-images.githubusercontent.com/204844/141966828-7ebc63af-0461-4e10-84ca-a4fbc37d4218.gif)

[Forum post](https://www.bryntum.com/forum/viewtopic.php?f=44&t=19475&p=96364#p96364)

Hello,

I found an issue while using the Bryntum Scheduler (from v.4.0.4 and forward) in vertical mode when trying to use  it wrapped in a defined height/width inside an Angular Material Dialog that includes more content and, because of that, there might be necessary some scrolling in order to use the Scheduler properly.

I've prepared and attached to this post a sample app (Angular v12, Material v12, Bryntum 4.3.2, although this same problem happens with v8 versions of Angular).

Basically, these are the reprosteps:
Open a Bryntum Scheduler inside a dialog;
Add an item in any position, with the info you like the most;
Try to drag the created item and move somewhere else:
   the item is moved in the correct position;
Try to scroll the dialog a little bit down;
Try again to drag the created item to move it somewhere else:
   the tooltip shows a different time on the position you're hovering the dragged item;
   the dropped item is positioned in an unexpected position;

Does anybody had this problem? It seems that a wrong value of the clientX is used to calculate the position of the scheduler, so the calculation executed when dropping an item gives an unexpected result. Is there any kind of workaround can you think of?

**Comment by taauntik** [comment]
Not reproducible anymore
